client/extensions/wp-super-cache/components/advanced/expiry-time.jsx

```diff
@@ -0,0 +1,180 @@
+	const renderCacheTimeout = () => {
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="cache_max_time">
+					{ translate( 'Cache Timeout' ) }
+				</FormLabel>
+
+				<FormTextInput
+					className="wp-super-cache__cache-timeout"
+					onChange={ handleChange( 'cache_max_time' ) }
+					value={ fields.cache_max_time || '' } />
+				{ translate( 'seconds' ) }
+				<FormSettingExplanation>
+					{
+						translate(
+							'How long should cached pages remain fresh? Set to 0 to disable garbage collection. ' +
+							'A good starting point is 3600 seconds.'
+						)
+					}
+				</FormSettingExplanation>
+			</FormFieldset>
+		);
+	};
+
+	const renderScheduler = () => {
+		return (
+			<FormFieldset className="wp-super-cache__scheduler">
+				<FormLabel>
+					{ translate( 'Scheduler' ) }
+				</FormLabel>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'interval' === fields.cache_schedule_type }
+						name="cache_schedule_type"
+						onChange={ handleRadio }
+						value="interval" />
+					<span>
+						{ translate( 'Timer' ) }
+						<FormTextInput
+							disabled={ 'interval' !== fields.cache_schedule_type }
+							onChange={ handleChange( 'cache_time_interval' ) }
+							value={ fields.cache_time_interval || '' } />
+						{ translate( 'seconds' ) }
+					</span>
+				</FormLabel>
+				<FormSettingExplanation isIndented>
+					{ translate( 'Check for stale cached files every interval seconds.' ) }
+				</FormSettingExplanation>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'time' === fields.cache_schedule_type }
+						name="cache_schedule_type"
+						onChange={ handleRadio }
+						value="time" />
+					<span>
+						{ translate( 'Clock' ) }
+						<FormTextInput
+							disabled={ 'time' !== fields.cache_schedule_type }
+							onChange={ handleChange( 'cache_scheduled_time' ) }
+							value={ fields.cache_scheduled_time || '' } />
+						{ translate( 'HH:MM' ) }
+					</span>
+				</FormLabel>
+				<FormSettingExplanation isIndented>
+					{ translate( 'Check for stale cached files at this time (UTC) or starting at this time ' +
+						'every interval below.' ) }
+				</FormSettingExplanation>
+			</FormFieldset>
+		);
+	};
+
+	const renderInterval = () => {
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="cache_schedule_interval">
+					{ translate( 'Interval' ) }
+				</FormLabel>
+
+				<FormSelect
+					id="cache_schedule_interval"
+					name="cache_schedule_interval"
+					onChange={ handleSelect }
+					value={ fields.cache_schedule_interval || 'five_minutes_interval' }>
+					<option value="five_minutes_interval">{ translate( 'Once every five minutes' ) }</option>
+					<option value="jetpack_sync_interval">{ translate( 'Every 5 minutes' ) }</option>
+					<option value="minutes_10">{ translate( 'Every 10 minutes' ) }</option>
+					<option value="minutes_30">{ translate( 'Every 30 minutes' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( '30 Minutes' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).